java/vector/src/main/java/org/apache/arrow/vector/LargeVarBinaryVector.java

```diff
@@ -112,6 +113,20 @@ public byte[] get(int index) {
     return result;
   }
 
+  /**
+   * Read the value at the given position to the given output buffer.
+   * The caller is responsible for checking for nullity first.
+   *
+   * @param index position of element.
+   * @param outputBuffer the buffer to write into.
+   */
+  public void read(int index, ReusableBuffer<?> outputBuffer) {
+    final long startOffset = getStartOffset(index);
+    final long dataLength =
+        offsetBuffer.getLong((long) (index + 1) * OFFSET_WIDTH) - startOffset;
```

Comment:
Use getEndOffset instead here rather than reimplement?

Reply:
getEndOffset() is not implemented for Large*Vectors. I can use it for Var*Vector and write an implementation for Large*Vectors.